services/user/Supervisor/ui/src/oauth/db.ts

```diff
@@ -0,0 +1,57 @@
+import { Result } from '../hostInterface';
+import { RecoverableErrorPayload } from '../plugin/errors';
+import { OAUTH_REQUEST_KEY } from '../constants';
+import { getSupervisor } from "@psibase/common-lib";
+import { z } from 'zod';
+
+export const supervisor = getSupervisor();
+
+export const OauthRequestSchema = z.object({
```

Comment:
`zOauthRequest`

Reply:
Parsing with this schema as is will pass with empty strings. Is that intentional or do you want to make a min length required?